war/src/main/js/util/jenkins.js

```diff
@@ -192,18 +192,22 @@ exports.loadTranslations = function(bundleName, handler, onError) {
 /**
  * Runs a connectivity test, calls handler with a boolean whether there is sufficient connectivity to the internet
  */
-exports.testConnectivity = function(handler) {
+exports.testConnectivity = function(siteId, handler) {
```

Comment:
I've moved the variable upstairs just to avoid magic constants on the library level.